doc/Homepage.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "HoloViews is a [Python](http://python.org) library that makes analyzing and visualizing scientific or engineering data much simpler, more intuitive, and more easily reproducible.  Without HoloViews, there are typically many steps required before you can see your data, whether you use a GUI interactively or write a function or script to build up a plot.  HoloViews instead lets you store your data in an annotated format that is instantly visualizable, with immediate access to both the numeric data *and* its visualization.  For instance, if you wrap a two-dimensional dataset like the fractal below ([mandelbrot.npy](https://github.com/ioam/holoviews/raw/master/doc/mandelbrot.npy)) in a HoloViews ``Image`` object named ``fractal``,  you can just type ``fractal`` to view it as an image in an [IPython/Jupyter Notebook](http://ipython.org/notebook/).  Most importantly, combining it with other objects is now easy -- you can e.g. view it annotated with a horizontal line and a histogram, next to a slice of it from the indicated cross-section, all without writing any plotting code:"
+    "HoloViews is a [Python](http://python.org) library that makes analyzing and visualizing scientific or engineering data much simpler, more intuitive, and more easily reproducible.  Without HoloViews, there are typically many steps required before you can see your data, whether you use a GUI interactively or write a function or script to build up a plot.  HoloViews instead lets you store your data in an annotated format that is instantly visualizable, with immediate access to both the numeric data *and* its visualization.  For instance, if you wrap a two-dimensional dataset like the fractal below ([mandelbrot.npy](https://github.com/ioam/holoviews/raw/master/doc/mandelbrot.npy)) in a HoloViews ``Image`` object named ``fractal``,  you can either type ``fractal.data`` to get at the underlying NumPy array, or just ``fractal`` to have it visualize itself automatically in an [IPython/Jupyter Notebook](http://ipython.org/notebook/).  Most importantly, combining the fractal image with other objects is now easy -- you can e.g. view it annotated with a horizontal line and a histogram, next to a slice of it from the indicated cross-section, all without writing any plotting code explicitly:"
    ]
```

Comment:
I agree that that aspect isn't coming through right now, but it's difficult to see how to make it clear.  People have a hard enough time understanding even the basic idea of how HoloViews works and how it's different from standard plotting APIs, and it's hard to see how one could convey that while also trying to convey that it can also be used to make visualizations of bits of some larger external data structure.  Strictly speaking, I think it does cover the Pandas and xarray cases already as written, in the sense that those objects remain accessible as .data even after wrapping, right?  It's just that it's a different workflow, where only part of the object is exposed as a visualization at a time, rather than having the entire collection of data each always individually and collectively visualizable.

If you can think of how to say it, maybe try editing it and pushing to this branch?